code/modules/wiremod/components/action/pathfind.dm

```diff
@@ -114,7 +127,10 @@
 		TIMER_COOLDOWN_END(parent, COOLDOWN_CIRCUIT_PATHFIND_SAME)
 
 		old_dest = destination
-		path = get_path_to(src, destination, max_range, id=path_id)
+		var/udv = TRUE
+		if(use_diag_val == 0) // I wish it didn't have to be this way
+			udv = FALSE
+		path = get_path_to(entity ||= src, destination, max_range, id=path_id, use_diags = udv)
```

Comment:
I know I commented on this before but `path = get_path_to(entity ||= src, destination, max_range, id=path_id, use_diags = udv)` would be more correct (didn't properly know what || did, but now I saw "The entire expression takes the value of the last argument to be evaluated.", My bad).